homeassistant/components/nut/const.py

```diff
@@ -91,6 +91,7 @@
         name="Load Restart Delay",
         native_unit_of_measurement=TIME_SECONDS,
         icon="mdi:timer-outline",
```

Comment:
I think icons should be removed  if a device class is set. But please verify that defaults look correct.

Reply:
I may be wrong but from my tests and search in the frontend repo this device_class has no icon set